### ISSUE TITLE
Switching between git repository don't show Repository PR Stats tab

### ISSUE THREAD

**Issue by bnchou** [jeffpriz/devops-pr-stats#11]
When you are in a specific repository, in the Repository PR Stats and then switch to another git repository with the combo box at the top.
![image](https://user-images.githubusercontent.com/7933861/117298075-38125980-ae77-11eb-9329-54189a2f76bd.png)
The tab selected go back to `Files` instead of staying on `Repository PR Stats` tab like other tabs.

**Comment by jeffpriz** [comment]
Yeah, I've felt that pain when swapping repos.. I will have to research in to the extension capabilities to see how to make it keep my hub selected
